hibike/devices/ExampleDevice/example_device.h

```diff
@@ -2,23 +2,33 @@
 #define EX_DEVICE_H
 
 #include "hibike_device.h"
+ 
+// You must replace EXAMPLE_DEVICE with the device id of the device you are writing firmware for
+// The device id's are defined in hibike/lib/hibike/devices.h
+// The device id should correspond to a device id in hibike/hibikeDevices.json
 
 //////////////// DEVICE UID ///////////////////
 hibike_uid_t UID = {
   EXAMPLE_DEVICE,                      // Device Type
-  0,                      // Year
+  1,                      // Year
   UID_RANDOM,     // ID
 };
 ///////////////////////////////////////////////
 
+#define NUM_PARAMS 6 //change to however many params are needed
```

Comment:
Where is `NUM_PARAMS` used? Based on the current diff it looks deprecated, in which case you should consider removing it.